src/script.js

```diff
@@ -18,6 +18,16 @@ var upCount = 0;
 var shouldSpeak = true;
 var storageItems= [];
 var storageArr = [];
+var save = document.getElementById("Save");
+var backUrl = localStorage.getItem("theValue");
+var box = document.getElementById("box");
+save.addEventListener("click",()=>{
+    box.style.backgroundImage = "url("+backUrl+")";
+    box.style.backgroundRepeat = "no-repeat";
+    box.style.backgroundSize = "cover";
+//    alert("Settings Applied Successfully");
```

Comment:
Comments must be added only in case of documentation.
Since there is none. Please remove the commented line.
Thanks!